index.yaml

```diff
@@ -307,7 +307,28 @@ entries:
   lacework-agent:
   - apiVersion: v2
     appVersion: "1.0"
-    created: "2022-09-22T15:51:55.030079-07:00"
+    created: "2022-09-23T10:22:23.281242768-07:00"
+    description: Lacework Agent
+    digest: 0b02e8db0d9bddd5576d0d1d9374fd9217c594d41026a0da37b35b47a9615364
+    home: https://www.lacework.com
+    icon: https://www.lacework.com/wp-content/uploads/2019/07/Lacework_Logo_color_2019.svg
+    keywords:
+    - monitoring
+    - security
+    - run-time
+    - metric
+    - troubleshooting
+    kubeVersion: '> 1.9.0-0'
+    maintainers:
+    - email: info@lacework.net
+      name: lacework-support
+    name: lacework-agent
+    urls:
+    - lacework-agent-6.0.1.tgz
+    version: 6.0.1
```

Comment:
Sorry if this a dump question. Should this much the agent release? Is this helm chart associated with the the 6.0.2 agent hot fix release or is this helm chart release separate?

Reply:
This is a Helm Chart only change -- we'll have another release with 6.0.2 hot fix.